app/models/Education.js

```diff
@@ -0,0 +1,102 @@
+const mongoose = require('mongoose');
+const CONSTANTS = require('../../config/constants');
+
+const Schema = mongoose.Schema;
+const educationSchema = new Schema(
+    {
+        secondary: {
+            year_of_comp: {
+                type: Number,
+            },
+            board: {
+                type: String,
+            },
+            performance: {
+                scale: {
+                    type: String,
+                    enum: [
+                        CONSTANTS.ENUMS.USER.PERFORMANCE_SCALE.PERCENTAGE,
+                        CONSTANTS.ENUMS.USER.PERFORMANCE_SCALE.CGPA
+                    ],
+                    default: CONSTANTS.ENUMS.USER.PERFORMANCE_SCALE.CGPA,
+                },
+                value: {
+                    type: Number,
+                },
+            },
+            school: {
+                type: String,
+            },
+            required: true,
```

Comment:
does this "required" tag conditions on all of inner attributes that they are all needed to be entered or need to put required in each of it ?????